storage/iordf/pom.xml

```diff
@@ -21,19 +21,12 @@
         <dependency>
             <groupId>org.apache.jena</groupId>
             <artifactId>jena-core</artifactId>
-            <version>2.7.4</version>
-            <exclusions>
-                <exclusion>
-                    <artifactId>slf4j-log4j12</artifactId>
-                    <groupId>org.slf4j</groupId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
-            <version>1.7.10</version>
-            <scope>test</scope>
+            <version>4.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.jena</groupId>
+            <artifactId>jena-arq</artifactId>
+            <version>4.2.0</version>
```

Comment:
![image](https://user-images.githubusercontent.com/348496/134131071-31084fdb-27e5-44ed-8ce6-cdfa411b82a1.png)

It looks like the latest version for java 1.8 is 3.9.0